src/features/Students/StudentsMetrics/index.jsx

```diff
@@ -2,28 +2,34 @@ import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { Card, CardGrid, ToggleButton } from '@edx/paragon';
 import { ToggleButtonGroup } from 'react-paragon-topaz';
-import { fetchMetricsData } from 'features/Students/data/thunks';
+import { fetchClassesMetricsData, fetchStudentsMetricsData } from 'features/Students/data/thunks';
 
 import 'features/Students/StudentsMetrics/index.scss';
 
 const StudentsMetrics = () => {
   const dispatch = useDispatch();
-  const stateMetrics = useSelector((state) => state.students.metrics.data);
+  const institution = useSelector((state) => state.main.selectedInstitution);
+  const studentsMetrics = useSelector((state) => state.students.studentsMetrics.data);
+  const classesMetrics = useSelector((state) => state.students.classesMetrics.data);
+  const days = 7; // Temporal variable
 
   useEffect(() => {
-    dispatch(fetchMetricsData());
-  }, [dispatch]);
+    if (Object.keys(institution).length > 0) {
+      dispatch(fetchStudentsMetricsData(institution.id, days));
+      dispatch(fetchClassesMetricsData(institution.id, days));
+    }
+  }, [institution, dispatch]);
 
   return (
     <div className="container-cards d-flex flex-column">
       <ToggleButtonGroup type="radio" defaultValue={1} name="period">
         <ToggleButton id="tbg-radio-1" value={1} variant="outline-primary">
           This week
         </ToggleButton>
-        <ToggleButton id="tbg-radio-2" value={2} variant="outline-primary">
+        <ToggleButton id="tbg-radio-2" value={2} variant="outline-primary" disabled>
```

Comment:
Add the inline comment "temporal" here as well. @sergivalero20

Reply:
@anfbermudezme It's not possible add inline comments there.

Reply:
@sergivalero20 Try this https://www.freecodecamp.org/news/html-comment-how-to-comment-out-a-line-or-tag-in-html/#:~:text=How%20to%20Write%20Inline%20Comments,tag%20won't%20be%20affected.

Reply:
That won't work in React,  @anfbermudezme , maybe a best approach could be do something like this:

```suggestion
        {/** Temporal */}
        <ToggleButton id="tbg-radio-2" value={2} variant="outline-primary" disabled>
```